csharp/src/Client/AdbcCommand.cs

```diff
@@ -29,28 +29,23 @@ public sealed class AdbcCommand : DbCommand
     {
         private AdbcStatement adbcStatement;
         private int _timeout = 30;
+        private bool _disposed;
```

Comment:
I will eventually file a work item to be consistent about formatting of field names across the project. This file was egregious in that it had one of each so there was no way to keep the file consistent... .